lib/components/form-fieldset.vue

```diff
@@ -27,12 +38,15 @@
                 return this.horizontal ? ('col-sm-' + (12 - this.labelSize)) : 'col-12';
             }
         },
-        mounted() {
-            const content = this.$refs.content;
-            if (!content) {
-                return;
+        methods: {
+            target() {
```

Comment:
Would maybe placing the target computation in `mounted()` and `updated()` be safer (and have target as a data prop)?